hoot-core/src/main/cpp/hoot/core/conflate/highway/HighwaySnapMerger.cpp

```diff
@@ -193,9 +193,6 @@ bool HighwaySnapMerger::_mergePair(const OsmMapPtr& map, ElementId eid1, Element
   OsmUtils::logElementDetail(e1, map, Log::Trace, "HighwaySnapMerger: e1");
   OsmUtils::logElementDetail(e2, map, Log::Trace, "HighwaySnapMerger: e2");
 
-  // This doesn't seem to always be true.
-  //assert(e1->getStatus() == Status::Unknown1);
-
```

Comment:
This was causing a debug failure sometimes and should be removed and not just commented out.